common/source/docs/common-omnibusnanov6.rst

```diff
@@ -50,10 +63,10 @@ Default UART order
 ==================
 
 - SERIAL0 = console = USB
-- SERIAL1 = Telemetry1 = USART1
-- SERIAL2 = Telemetry2 = USART4
+- SERIAL1 = Telemetry1 = USART1 (only available on original V6 version)
+- SERIAL2 = Telemetry2 = USART4 (RX only for ESC telemetry)
 - SERIAL3 = GPS1 = USART6
-- SERIAL4 = not assigned
+- SERIAL4 = GPS2 = USART3 available in BRD_ALT_CONFIG = 1 on RX3 / TX3 pins (I2C in default config)
```

Comment:
RX3/TX3 pins comment redundant , jsut UASRT3 available in BRD_ALT_CONFIG = 1 is enough...also hot link that param...

Reply:
fixed